catalyst/utils/distributed.py

```diff
@@ -0,0 +1,237 @@
+from collections import OrderedDict
+import copy
+import inspect
+import os
+import socket
+import subprocess
+import sys
+from typing import Dict, Tuple
+
+import torch
+from torch import nn
+import torch.distributed
+
+from catalyst import utils
+from catalyst.utils.tools.typing import (
+    Criterion, Device, Model, Optimizer, Scheduler
+)
+
+
+def get_rank() -> int:
+    """
+    If torch.distributed is initialized returns ``rank``, otherwise ``-1``
+    :return:
+    """
```

Comment:
Could you please use Google style format for the docstrings?

Reply:
done